### PR TITLE
[ADD] product_priority

### PR DESCRIPTION
**Pull request by bodedra**
#322

### CODE REVIEW

File: product_priority/__init__.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Copyright (C) 2017 - TODAY, Open Source Integrators
```
Comment (max3903) [inline product_priority/__init__.py]: @bodedra `2018`

**Review by max3903 — CHANGES_REQUESTED** [review]

**Review by max3903 — APPROVED** [review]

**Comment by JordiBForgeFlow** [comment]
Why would you assign the priority to product.template and not product.product? As I see it, you might want to ship black t-shirt as a lower priority than yellow t-shirt when someone order the t-shirt that has 2 variants

**Comment by max3903** [comment]
@jbeficent We need to order 2 different product templates, not product variants.

If the demand is for a small black 2018 t-shirt, you can't ship a large yellow 2018 one, but we could ship the small black 2017 t-shirt.

**Review by JordiBForgeFlow — APPROVED** [review]

**Comment by JordiBForgeFlow** [comment]
Thanks for the clarification @max3903

**Review by guewen — APPROVED** [review]

**Pull request merged**